packages/calling-stateful-client/src/CallDeclarativeCommon.ts

```diff
@@ -3,7 +3,7 @@
 
 import { CallContext } from './CallContext';
 import { CallCommon } from './BetaToStableTypes';
-/* @conditional-compile-remove(close-captions) */
+/* @conditional-compile-remove(close-captions) */ /* @conditional-compile-remove(call-transfer) */
```

Comment:
not impacting this PR but we need to make sure that the import of this item has all the right CC statements on it. Stabilizing features will require this and the engineer working through this will be a headache. good job here just wanted to call that out!